### ISSUE TITLE
Node Odinbook: Remove mention of express-generator

### ISSUE THREAD

**Issue by MaoShizhong** [TheOdinProject/curriculum#28875]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

We no longer even mention express-generator in the course.

## Acceptance criteria

- [ ] Step 3 "Start the app however you like..." is removed entirely from the assignment.

Please comment below if you'd like to be assigned to this issue. Please do not open a PR unless you have been assigned by a maintainer.

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-nodejs-odin-book

**Comment by devnchill** [comment]
I'll be glad to help 🙂

**Comment by MaoShizhong** [comment]
All yours

**Comment by devnchill** [comment]
woah i created a pr and that got detected and mentioned in this issue nice

**Comment by XAJX179** [comment]
> woah i created a pr and that got detected and mentioned in this issue nice

nothing got auto detected you mentioned closing this issue in your PR

**Comment by devnchill** [comment]
oh you mean in my pr where i mentioned #XXXX in issue no ...
